apps/dashboard/config/configuration_singleton.rb

```diff
@@ -368,6 +368,11 @@ def ood_bc_card_time
     (ood_bc_card_time_int < 0) ? 0 : ood_bc_card_time_int
   end
 
+  def sessions_poll_delay
+    poll_delay = ENV['POLL_DELAY']
+    poll_delay.nil? ? config.fetch(:sessions_poll_delay, '10000') : poll_delay.to_s
+  end
```

Comment:
I have a couple things here.

First can we prepend `bc_` to this. That'll follow other configuration names so folks can see they all relate to batch_connect.

Secondly - I'm thinking we need some sort of check like casting `to_i` to be sure it's an integer. I also think that 10000 should be the default and likely minimum too. I'd hate to see someone configure `1` and just blow up their system lol.

Reply:
Yes, that is better. I completed the updates